siteimprove/admin/js/siteimprove.js

```diff
@@ -133,20 +165,43 @@
       return result;
     };
 
+  function getDocumentHeight() {
+    const body = document.body,
+    html = document.documentElement;
+
+    const height = Math.max( body.scrollHeight, body.offsetHeight, 
+    html.clientHeight, html.scrollHeight, html.offsetHeight );
+
+    return height;
+  }
+
+  function getCorrectDocument (si_prepublish_data) {
+    const newDiv = document.createElement("div"); 
+    newDiv.setAttribute("id","div_iframe"); 
+    document.body.appendChild(newDiv);
+    newDiv.innerHTML = "<iframe id='domIframe' src="+ si_prepublish_data.url.concat("&si_preview=1") +" style='height:"+getDocumentHeight()+"px;width:100%'></iframe>";
```

Comment:
Is `100vh` not better than the calculation you do above?

Reply:
Can you test whether it has the same output?

Reply:
The output is almost the same, but directly speaking, for our needs the results are the same.

Changed to `100vh`